app.rb

```diff
@@ -0,0 +1,88 @@
+require_relative 'lib/database_connection'
+require 'sinatra/base'
+require 'sinatra/reloader'
+require_relative 'lib/peep_repository'
+require_relative 'lib/user'
+require_relative 'lib/user_repository'
+
+DatabaseConnection.connect('chitter_test')
```

Comment:
Is this 'chitter_test' the 'real' database, or is it supposed to be for the RSpec tests?
If for tests, I think it should be in the spec files